packages/suite/src/views/suite/SwitchDevice/DeviceItem/DeviceDetail.tsx

```diff
@@ -0,0 +1,27 @@
+import { spacingsPx, typography } from '@trezor/theme';
+import { ReactNode } from 'react';
+import styled from 'styled-components';
+
+const Container = styled.div`
+    display: flex;
+    flex: 1;
+    flex-direction: column;
+    overflow: hidden;
+    align-self: center;
+`;
+
+const DeviceLabel = styled.div`
+    ${typography.body};
+    margin-bottom: -${spacingsPx.xxs};
+    min-width: 0;
+    color: ${({ theme }) => theme.textDefault};
+    overflow: hidden;
+    text-overflow: ellipsis;
```

Comment:
You might use `TruncateWithTooltip` here

Reply:
:heavy_check_mark: